### ISSUE TITLE
"deflate": is this zlib format or raw format?

### ISSUE THREAD

**Issue by ricea** [ricea/compressstream-explainer#8]
It's ambiguous whether the "deflate" format is raw or includes the zlib header and footer. It might be good to rename it--maybe "deflate-raw".

**Comment by ericlaw1979** [comment]
Yeah, I think it makes the most sense to match HTTP Content-Encoding and have 'deflate' mean Zlib-wrapped DEFLATE ([RFC1950](https://www.ietf.org/rfc/rfc1950.txt))

Node uses '[deflateRaw](https://nodejs.org/api/zlib.html#zlib_class_zlib_deflateraw)' for the bare format and this sounds reasonable.

**Comment by ricea** [comment]
@ericlaw1979 Thanks. We settled on using "deflate" to mean "zlib" for consistency with HTTP, and will add "deflate-raw" in future to support zip and other use cases where the header and footer are unwanted.
